baseStats.json

```diff
@@ -321,13 +321,12 @@
       "evolvesTo": 0,
       "evolvesFrom": "Kakuna",
       "quickMoves": [
-        "Bug Bite",
-        "Poison Jab"
+        201,
+        224
       ],
       "cinematicMoves": [
-        "Sludge Bomb",
-        "Aerial Ace",
-        "X Scissor"
+        90,
+        45
```

Comment:
I am not sure if this is on purpose and I may be wrong, but it seems an attack is missing in line 330. There are also missing attacks in lines: 1027, 1050, 2751 and 2840.

Sorry if I am mistaken, I was trying to learn from reading code and just wanted to help.